[package] doc string cleanup extravaganza

lobstergrindset opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#59843 [package] doc string cleanup extravaganza**

Differential Revision: [D29049342](https://our.internmc.facebook.com/intern/diff/D29049342)

Commit:
[package] doc string cleanup extravaganza

[ghstack-poisoned]

Lilyjjo commented:
@Lilyjjo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29049342).

suo commented:
I'll let @SplitInfinity review this one as he mentioned he had some RST changes as well.

Commit:
Update on "[package] doc string cleanup extravaganza"


Differential Revision: [D29049342](https://our.internmc.facebook.com/intern/diff/D29049342)

[ghstack-poisoned]

Lilyjjo commented:
@Lilyjjo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29049342).
